---
title: "Modbed track: Visualization of modified bases in single-molecule sequencing"
date: 2023-12-06
featured: true
draft: false
link: http://doi.org/10.1016/j.xgen.2023.100455
weight: 42
---

[Learn more](http://doi.org/10.1016/j.xgen.2023.100455)
{{< load-photoswipe >}}
{{< figure src = "https://wang.wustl.edu/img/Daofeng_resized_cellgenomics_23.png" >}}

# Abstract:

Recent advances in long-read sequencing technologies have not only dramatically increased sequencing read length but also have improved the accuracy of detecting chemical modifications to the canonical nucleotide bases, thus opening exciting venues to investigate the epigenome. Currently, the ability to visualize modified bases from long-read sequencing data in genome browsers is still limited, preventing users from easily and fully exploring these type of data. To address this limitation, the WashU Epigenome Browser introduces the modbed track type, which provides visualization of modification details in each single read as well as aggregated modifications of individual or multiple molecules across a dynamic range of resolutions. The modbed file can be uploaded for visualization as a local track or viewed with an accessible URL freely on the WashU Epigenome Browser at https://epigenomegateway.wustl.edu/.
