---
title: "Regulatory transposable elements in theencyclopedia of DNA elements"
date: 2024-08-31
featured: true
draft: false
link: https://doi.org/10.1038/s41467-024-51921-6
weight: 46
---

[Learn more](https://doi.org/10.1038/s41467-024-51921-6)
{{< load-photoswipe >}}
{{< figure src = "https://wang.wustl.edu/img/Alan_Nature_Communication_2024.png" >}}

# Abstract:

Transposable elements (TEs) comprise ~50% of our genome, but knowledge of how TEs affect genome evolution remains incomplete. Leveraging ENCODE4data, we provide the most comprehensive study to date of TE contributions to the regulatory genome. We find 236,181 (~25%) human candidate cis-regulatory elements (cCREs) are TE-derived, with over 90% lineage-specific since the human-mouse split, accounting for 8–36% of lineage-specific cCREs.Except for SINEs, cCRE-associated transcription factor (TF) motifs in TEs are derived from ancestral TE sequence more than expected by chance. We show that TEs may adopt similar regulatory activities of elements near their integration site. Since human-mouse divergence, TEs have contributed 3–56% of TF binding site turn over events across 30 examined TFs. Finally, TE-derived cCREs are similar to non-TE cCREs in terms of MPRA activity and GWAS variant enrichment.Overall, our results substantiate the notion that TEs have played an important role in shaping the human regulatory genome.
