---
title: "Regenerating zebrafish fin epigenome is characterized by stable lineage-specific DNA methylation and dynamic chromatin accessibility"
featured: true
draft: false
link: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-1948-0
weight: 16
---

[Learn more](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-1948-0)
{{< load-photoswipe >}}
{{< figure src = "http://wang.wustl.edu/img/slide20.png" >}}

# Abstract:
Background:   
Zebrafish can faithfully regenerate injured fins through the formation of a blastema, a mass of proliferative cells that can grow and develop into the lost body part. After amputation, various cell types contribute to blastema formation, where each cell type retains fate restriction and exclusively contributes to regeneration of its own lineage. Epigenetic changes that are associated with lineage restriction during regeneration remain underexplored.
\
\
Results:  
We produce epigenome maps, including DNA methylation and chromatin accessibility, as well as transcriptomes, of osteoblasts and other cells in uninjured and regenerating fins. This effort reveals regeneration as a process of highly dynamic and orchestrated transcriptomic and chromatin accessibility changes, coupled with stably maintained lineage-specific DNA methylation. The epigenetic signatures also reveal many novel regeneration-specific enhancers, which are experimentally validated. Regulatory networks important for regeneration are constructed through integrative analysis of the epigenome map, and a knockout of a predicted upstream regulator disrupts normal regeneration, validating our prediction.
\
\
Conclusion:  
Our study shows that lineage-specific DNA methylation signatures are stably maintained during regeneration, and regeneration enhancers are preset as hypomethylated before injury. In contrast, chromatin accessibility is dynamically changed during regeneration. Many enhancers driving regeneration gene expression as well as upstream regulators of regeneration are identified and validated through integrative epigenome analysis.