---
title: "Genetic, epigenetic, and environmental mechanisms govern allele-specific gene expression"
date: 2018-11-28T15:14:39+10:00
featured: true
draft: false
link: https://genome.cshlp.org/content/32/6/1042
weight: 3
---

[Learn more](https://genome.cshlp.org/content/32/6/1042)
{{< load-photoswipe >}}
{{< figure src = "http://wang.wustl.edu/img/slide34.png" >}}

# Abstract: 

Allele-specific expression (ASE) is a phenomenon in which one allele is preferentially expressed over the other. Genetic and epigenetic factors cause ASE by altering the final composition of a gene's product, leading to expression imbalances that can have functional consequences on phenotypes. Environmental signals also impact allele-specific expression, but how they contribute to this cross talk remains understudied. Here, we explored how genotype, parent-of-origin, tissue, sex, and dietary fat simultaneously influence ASE biases. Male and female mice from a F1 reciprocal cross of the LG/J and SM/J strains were fed a high or low fat diet. We harnessed strain-specific variants to distinguish between two ASE classes: parent-of-origin-dependent (unequal expression based on parental origin) and sequence-dependent (unequal expression based on nucleotide identity). We present a comprehensive map of ASE patterns in 2853 genes across three tissues and nine environmental contexts. We found that both ASE classes are highly dependent on tissue and environmental context. They vary across metabolically relevant tissues, between males and females, and in response to dietary fat. We also found 45 genes with inconsistent ASE biases that switched direction across tissues and/or environments. Finally, we integrated ASE and QTL data from published intercrosses of the LG/J and SM/J strains. Our ASE genes are often enriched in QTLs for metabolic and musculoskeletal traits, highlighting how this orthogonal approach can prioritize candidate genes. Together, our results provide novel insights into how genetic, epigenetic, and environmental mechanisms govern allele-specific expression, which is an essential step toward deciphering the genotype-to-phenotype map.

