---
title: "Transcript assembly improves expression quantification of transposable elements in single-cell RNA-seq data"
featured: true
draft: false
link: https://genome.cshlp.org/content/early/2020/12/21/gr.265173.120.abstract
weight: 10
---

[Learn more](https://genome.cshlp.org/content/early/2020/12/21/gr.265173.120.abstract)
{{< load-photoswipe >}}
{{< figure src = "http://wang.wustl.edu/img/slide26.png" >}}

# Abstract: 

Transposable elements (TEs) are an integral part of the host transcriptome. TE-containing noncoding RNAs (ncRNAs) show considerable tissue specificity and play important roles during development, including stem cell maintenance and cell differentiation. Recent advances in single-cell RNA-seq (scRNA-seq) revolutionized cell typeâ€“specific gene expression analysis. However, effective scRNA-seq quantification tools tailored for TEs are lacking, limiting our ability to dissect TE expression dynamics at single-cell resolution. To address this issue, we established a TE expression quantification pipeline that is compatible with scRNA-seq data generated across multiple technology platforms. We constructed TE-containing ncRNA references using bulk RNA-seq data and showed that quantifying TE expression at the transcript level effectively reduces noise. As proof of principle, we applied this strategy to mouse embryonic stem cells and successfully captured the expression profile of endogenous retroviruses in single cells. We further expanded our analysis to scRNA-seq data from early stages of mouse embryogenesis. Our results illustrated the dynamic TE expression at preimplantation stages and revealed 146 TE-containing ncRNA transcripts with substantial tissue specificity during gastrulation and early organogenesis.