---
title: "Co-opted transposons help perpetuate conserved higher-order chromosomal structures"
featured: true
draft: false
link: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1916-8
weight: 17
---

[Learn more](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1916-8)
{{< load-photoswipe >}}
{{< figure src = "http://wang.wustl.edu/img/slide19.png" >}}

# Abstract:
Background:  
Transposable elements (TEs) make up half of mammalian genomes and shape genome regulation by harboring binding sites for regulatory factors. These include binding sites for architectural proteins, such as CTCF, RAD21, and SMC3, that are involved in tethering chromatin loops and marking domain boundaries. The 3D organization of the mammalian genome is intimately linked to its function and is remarkably conserved. However, the mechanisms by which these structural intricacies emerge and evolve have not been thoroughly probed.
\
\
Results:  
Here, we show that TEs contribute extensively to both the formation of species-specific loops in humans and mice through deposition of novel anchoring motifs, as well as to the maintenance of conserved loops across both species through CTCF binding site turnover. The latter function demonstrates the ability of TEs to contribute to genome plasticity and reinforce conserved genome architecture as redundant loop anchors. Deleting such candidate TEs in human cells leads to the collapse of conserved loop and domain structures. These TEs are also marked by reduced DNA methylation and bear mutational signatures of hypomethylation through evolutionary time.
\
\
Conclusions:  
TEs have long been considered a source of genetic innovation. By examining their contribution to genome topology, we show that TEs can contribute to regulatory plasticity by inducing redundancy and potentiating genetic drift locally while conserving genome architecture globally, revealing a paradigm for defining regulatory conservation in the noncoding genome beyond classic sequence-level conservation.